simple_locations/admin.py

```diff
@@ -27,11 +28,27 @@ class AreaTypeAdmin(admin.ModelAdmin):
     list_display = ('slug', 'name')
 
 
-class AreaAdmin(MPTTModelAutocompleteAdmin):
-    list_display = ( 'name', 'kind', 'location', 'code')
+class AreaChildrenInline(admin.TabularInline):
+    model = Area
+    fields = ['name', 'code', 'kind']
+    show_change_link = True
+    extra = 0
+
+
+class AreaAdmin(MPTTModelAutocompleteAdmin, admin.OSMGeoAdmin):
+    # default_lon = getattr(settings, 'GIS_DEFAULT_LAT', -8.8742)
```

Comment:
why this change?

Reply:
Because I'd like to see a more detailed map if I even go on to add coordinates from our geojson files/tempu.tl database ^^;